add_events.py

```diff
@@ -112,8 +112,12 @@ def main():
                 replaceHour += int(startHour)
             event = {}
             # Currently there are no labs in `subjects.json`
-            if (data[day][time][0] in subjects.keys()):
-                event['summary'] = "Class of " + subjects[data[day][time][0]].title()
+            # if (data[day][time][0] in subjects.keys()):
+            if (data[day][time][2] < 3):
```

Comment:
1. Do NOT do this. Remember that each course has an L-T-P designation. If this designation is of the form 0-0-N, then it is an N hour lab. This N can be 2, 3, 4, etc.

@athityakumar Fix this, please.